src/wrapper/jnienv.rs

```diff
@@ -165,8 +167,7 @@ impl<'a> JNIEnv<'a> {
         S: Into<JNIString>,
```

Comment:
I think this documentation must mention that the default behaviour of using FindClass can be overridden and reference the method that allows to do that.